src/graphics/device.js

```diff
@@ -1258,6 +1258,18 @@ pc.extend(pc, function () {
             }
         },
 
+        onVertexBufferDeleted: function () {
```

Comment:
No. It flushes some states on the device, because there were cases when some deleted buffer remained bound.